samples/openthread/cli/README.rst

```diff
@@ -100,6 +100,8 @@ The following snippets are available:
 * ``logging`` - Enables logging using RTT.
   For additional options, refer to :ref:`RTT logging <ug_logging_backends_rtt>`.
 * ``debug`` - Enables debugging the Thread sample with GDB thread awareness.
+* ``diag_gpio`` - Configures DK's Buttons and LEDs for diagnostic GPIO commands.
+  For more information, see `OpenThread Factory Diagnostics Module Reference <https://github.com/openthread/openthread/blob/main/src/core/diags/README.md>`_.
```

Comment:
Can we add hyperlinks into .rst directly, or we should put it into `doc/nrf/links.txt` and use ref here?

Reply:
`links.txt`, please. Otherwise looks good to me.

Reply:
sure, moved to `links.txt`